setuptools/command/upload_docs.py

```diff
@@ -59,6 +59,7 @@ def initialize_options(self):
         self.target_dir = None
 
     def finalize_options(self):
+        log.warn("Upload_docs command is deprecated. Use RTD instead.")
```

Comment:
```suggestion
        log.warn("Upload_docs command is deprecated. Use Read the Docs (https://readthedocs.org) instead.")
```

Fly-by suggestion: Let's include more context on what "RTD" is, for people who may not be familiar with it?

Reply:
Good suggestion. I worry accepting it will break the tests, but let's give it a try.